plugin/evm/vm_test.go

```diff
@@ -766,13 +765,11 @@ func testReorgProtection(t *testing.T, scheme string) {
 	// with the preferred chain lower than the last finalized block)
 	// should NEVER happen. However, the VM defends against this
 	// just in case.
-	if err := vm1.SetPreference(context.Background(), vm1BlkC.ID()); !strings.Contains(err.Error(), "cannot orphan finalized block") {
-		require.NoError(t, err, "Unexpected error when setting preference that would trigger reorg")
-	}
+	err = vm1.SetPreference(context.Background(), vm1BlkC.ID())
+	require.ErrorContains(t, err, "cannot orphan finalized block", "Expected error when setting preference that would orphan finalized block")
 
-	if err := vm1BlkC.Accept(context.Background()); !strings.Contains(err.Error(), "expected accepted block to have parent") {
-		require.NoError(t, err, "Unexpected error when setting block at finalized height")
-	}
+	err = vm1BlkC.Accept(context.Background())
+	require.ErrorContains(t, err, "expected accepted block to have parent", "Expected error when accepting orphaned block")
```

Comment:
defining sentinel error requires modifying geth code which I don't really want to do.